platforms/models.py

```diff
@@ -25,10 +25,9 @@ class Word(models.Model):
     def __str__(self):
         return self.word
 
-class UsersWords(models.Model):
-    user_word = models.ForeignKey(Word, on_delete=models.CASCADE, related_name='users_words')
-    user_name = models.ForeignKey(User, on_delete=models.CASCADE, related_name='users_words')
-    time = models.DateTimeField(auto_now_add=True)
+class UserWord(models.Model):
+    word = models.ForeignKey(Word, on_delete=models.CASCADE, related_name='users_words')
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='users_words')
+    date_add_word = models.DateTimeField()
```

Comment:
not add :) we going to use this field when a user successfully learned the word.
we use date format to know when exactly we learned 

1. field name learned_at or some other options
2. date without time - better
3. allow to keep this field empty